pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -142,9 +143,16 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, includeUniniti
 			continue
 		}
 
+		// Check for new RBAC RoleBinding first
+		if roleBinding, ok := list.Objects[i].(*rbac.RoleBinding); ok {
```

Comment:
Change this block to use a type switch since that what we are doing here with mutually exclusive `if` statements.